aspnet/fundamentals/caching/sample/test/CachingSample.Tests/project.json

```diff
@@ -0,0 +1,32 @@
+﻿{
+  "version": "1.0.0-*",
+  "description": "CachingSample.Tests Class Library",
+  "authors": [ "stevesmith" ],
+  "tags": [ "" ],
+  "projectUrl": "",
+  "licenseUrl": "",
+
+  "frameworks": {
+    "dnx451": { },
+    "dnxcore50": {
+      "dependencies": {
+        "Microsoft.CSharp": "4.0.1-beta-23409",
```

Comment:
A stand-alone library might need them, but you pull most of them in transitively.